File: Praktikum Pemrograman 1/Praktikum Pemrograman 1 modul 5/Python/PRAK505-2410817110006-HamkaArifani.py
def Biodata(Tahunlahir,Nama,Asal):
    TahunSekarang=2020
    umur=TahunSekarang-Tahunlahir
    angkatan=TahunSekarang
    print("Perkenalkan Nama Saya : ",Nama)
    print("Umur Saya : ",umur)
    print("Saya Adalah Angkatan : ", angkatan)
    print("Asal Saya Dari : ", Asal)
tahunLahir=int(input(""))
Namaku=input("")
Asal=input("")
Biodata(tahunLahir, Namaku, Asal)

File: Praktikum Pemrograman 1/Praktikum Pemrograman 1 modul 5/C/PRAK505-2410817110006-HamkaArifani.c
#include <stdio.h>
void Biodata(int tahunlahir, char Nama[], char Asal[]){
int tahunsekarang = 2020;
int umur=tahunsekarang-tahunlahir;
int angkatan=tahunsekarang;
printf("Perkenalkan Nama Saya : %s\n",Nama);
printf("Umur Saya : %d\n",umur);
printf("Saya Adalah Angkatan : %d\n",tahunsekarang);
printf("Asal Saya Dari : %s\n",Asal);
}
int main() {
int tahunLahir;
char A[20], B[15];
scanf(" %d",&tahunLahir);
scanf(" %[^\n]%*c",&A);
scanf(" %[^\n]%*c",&B);
Biodata(tahunLahir, A, B);
return 0;
}